src/components/Book.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import './Book.css';
+/* eslint-disable */
+
+const Book = (book) => {
+  const dispatch = useDispatch();
+  const { id, author, title, category } = book;
+
+  const handleDelte = () => {
+    dispatch(removeBook(id));
+  };
+
+  const Chapter = Math.round(Math.random() * 10);
+
+  return (
+    <div className="card">
+      <div className="dynamic-part">
+        <p className="category">{category}</p>
+        <p className="title">{title}</p>
+        <p className="author">{author}</p>
+        <div className="button-d">
+          <button type="button" className="button-click">
+            Comments
+          </button>
+          |
+          <button type="button" className="button-click" onClick={handleDelte}>
+            Remove Book
+          </button>
+          |
+          <button type="button" className="button-click">
+            Edit
+          </button>
+        </div>
```

Comment:
- [ ] Kindly check the styling for this section. It does not look like the design that is provided in Zeplin.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/202713003-e8bf53e2-172e-4344-ae96-96b0b4aed0ef.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202713081-50ed1132-5249-4bc4-8eb4-05531014dae2.png)